apps/token/contracts/Token.sol

```diff
@@ -21,6 +23,8 @@ contract Token is ERC20Token, AragonApp {
   }
 
   function mintFor(address contributorAccount, uint256 amount, uint32 contributionId) public isInitialized auth(MINT_TOKEN_ROLE) {
+    require(amount > 0, ERROR_INVALID_AMOUNT);
+
```

Comment:
@bumi is that cool ?

Reply:
just saw that I haven't done it everywhere. I thought about something like this: https://github.com/67P/kredits-contracts/blob/master/apps/contribution/contracts/Contribution.sol#L181

as that full string increases the contract and thus the deployment costs, afaik? so I thought it is better to use small strings.

Reply:
ahh okaay (y)